src/ansible/roles/packages/tasks/Ubuntu.yml

```diff
@@ -49,6 +49,13 @@
       - realmd
       - sssd
       - sssd-*
+  - name: Install packages required for passkey testing
+    apt:
+      state: present
+      name:
+      - gcc
+      - libssl-dev
+      - umockdev
```

Comment:
when: passkey_support to be consistent with Fedora?

Reply:
Done